homeassistant/components/sensor/geizhals.py

```diff
@@ -80,7 +80,10 @@ def icon(self):
     @property
     def state(self):
         """Return the best price of the selected product."""
-        return self._device.prices[0]
+        if not self._device.prices:
+            return STATE_UNKNOWN
```

Comment:
Return `None`.

Reply:
As far as I remember was it use to display the string `unknown` in the frontend but now the frontend can handle it directly.